### PR TITLE
SAC and AC identity

### PR DESCRIPTION
**Pull request by purujitgoyal**

### CODE REVIEW

File: openreview/venue_request/venue_request.py [hunk 1]
```diff
@@ -814,10 +814,10 @@ def setup_request_form(self):
                     'Assigned Senior Area Chair',
                     'All Area Chairs',
                     'Assigned Area Chair',
-                    'All Reviewers',
-                    'Assigned Reviewers'
+                    # 'All Reviewers',
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:817]: why did you comment these options?

File: openreview/venue_request/venue_request.py [hunk 2]
```diff
@@ -828,15 +828,15 @@ def setup_request_form(self):
                     'Program Chairs',
                     'All Senior Area Chairs',
                     'Assigned Senior Area Chair',
-                    'All Area Chairs',
-                    'Assigned Area Chair',
-                    'All Reviewers',
-                    'Assigned Reviewers'
+                    # 'All Area Chairs',
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:831]: Why did you comment these options?

File: openreview/venue_request/venue_request.py [hunk 3]
```diff
@@ -828,15 +828,15 @@ def setup_request_form(self):
...
+                    # 'Assigned Area Chair',
+                    # 'All Reviewers',
+                    # 'Assigned Reviewers'
                 ],
                 'default': ['Program Chairs'],
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:836]: you need to add the Assigned Senior Area Chair here too

File: openreview/venue_request/venue_request.py [hunk 4]
```diff
@@ -828,15 +828,15 @@ def setup_request_form(self):
...
                 'order': 31,
                 'required': False,
-                'hidden': True
+                # 'hidden': True
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:839]: can you remove this line?

File: tests/test_venue_request.py [hunk 5]
```diff
@@ -899,7 +900,7 @@ def test_venue_review_stage(self, client, test_client, selenium, request_page, h
 
         reviewer_client = openreview.Client(username='venue_reviewer2@mail.com', password='1234')
         reviewer_group = client.get_group('{}/Reviewers'.format(venue['venue_id']))
-        assert reviewer_group and len(reviewer_group.members) == 2
+        assert reviewer_group and len(reviewer_group.members) == 3
```
Comment (melisabok) [inline tests/test_venue_request.py:903]: I'm not sure why you had to change this assertion, are there more reviewers now?

**Comment by melisabok** [comment]
In order to test these changes you need to try to get the paper SAC groups and paper AC groups and check the SAC is readers of that group. 

For example, for the SAC group for paper 1: ICLR.cc/2022/Conference/Paper1/Senior_Area_Chairs must have as readers the PC groups and the ICLR.cc/2022/Conference/Paper1/Senior_Area_Chairs group so the member of the group can read their own groups.

You have to check something similar to AC groups, like ICLR.cc/2022/Conference/Paper1/Area_Chairs

File: openreview/venue_request/venue_request.py (hunk 4 above)
Comment (purujitgoyal) [reply openreview/venue_request/venue_request.py:839]: Still keep it hidden?

File: openreview/venue_request/venue_request.py (hunk 3 above)
Comment (melisabok) [reply openreview/venue_request/venue_request.py:836]: I think Celeste's issue wants to have this field not hidden to the user can select the identity options but I do think the default value must contain the assigned SAC, if the users forgets to select it then the SAC can not see their own assignments.

File: openreview/venue_request/venue_request.py (hunk 1 above)
Comment (purujitgoyal) [reply openreview/venue_request/venue_request.py]: Added them back.

File: openreview/venue_request/venue_request.py (hunk 2 above)
Comment (purujitgoyal) [reply openreview/venue_request/venue_request.py]: Added them back.

File: openreview/venue_request/venue_request.py (hunk 4 above)
Comment (purujitgoyal) [reply openreview/venue_request/venue_request.py]: Done.

File: openreview/venue_request/venue_request.py (hunk 3 above)
Comment (purujitgoyal) [reply openreview/venue_request/venue_request.py]: Done.

File: tests/test_venue_request.py (hunk 5 above)
Comment (purujitgoyal) [reply tests/test_venue_request.py]: it was showing some error on my local. reverted the change.

File: openreview/venue_request/venue_request.py [hunk 6]
```diff
@@ -817,7 +817,7 @@ def setup_request_form(self):
                     'All Reviewers',
                     'Assigned Reviewers'
                 ],
-                'default': ['Program Chairs'],
+                'default': ['Program Chairs', 'Assigned Senior Area Chair'],
                 'order': 31,
                 'required': False,
                 'hidden': True
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:823]: could you also remove this line? so the PCs can select the area chair identity?

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**